Desktop/components/JASP/Widgets/FileMenu/PrefsUI.qml

```diff
@@ -185,15 +185,85 @@ ScrollView
 			DropDown
 			{
 				id:							languages
-				label:						qsTr("Choose language  ")
+				label:						qsTr("Choose language")
 				source:						languageModel
 				startValue: 				languageModel.currentLanguage
 				onValueChanged: 			languageModel.currentLanguage = value
 
-				KeyNavigation.tab: 			altnavcheckbox
+				KeyNavigation.tab: 			useAlternativeLocale
 				
 			}
 			
+			CheckBox
+			{
+				id:							useAlternativeLocale
+				label:						qsTr("Use an alternative locale for numbers")
+				checked:					languageModel.useAlternativeLocale
+				onCheckedChanged:			languageModel.useAlternativeLocale = checked
+				toolTip:					qsTr("Use the locale specified below for display of numbers and currency.") //dates and times will have to be added later
+				
+				KeyNavigation.tab:			alternativeLocaleTerritory
+			}
+			
+			RowLayout
+			{
+				spacing:		jaspTheme.generalAchorMargin
+				
+				Group
+				{
+					enabled:					languageModel.useAlternativeLocale
+					
+					DropDown
+					{
+						id:							alternativeLocaleLanguage
+						label:						qsTr("Alt. language")
+						values:		 				languageModel.altLanguages
+						startValue:				 	languageModel.currentAltLanguage
+						onValueChanged:				if(value != "") languageModel.currentAltLanguage = value
+						addEmptyValue:				false
+						KeyNavigation.tab:			alternativeLocaleTerritory
+					}
+					
+					DropDown
+					{
+						id:							alternativeLocaleTerritory
+						label:						qsTr("Alt. territory")
+						values:		 				languageModel.altTerritories
+						startValue:				 	languageModel.currentAltTerritory
+						value:						languageModel.currentAltTerritory
+						onValueChanged:				if(value != "") languageModel.currentAltTerritory = value
+						addEmptyValue:				false
+						KeyNavigation.tab:			altnavcheckbox
+					}
+					
+				}
+				
+				Item{ width: jaspTheme.generalAchorMargin}
+				
+				Rectangle
+				{
+					implicitWidth:		exampleFormattingText.implicitWidth
+					implicitHeight:		exampleFormattingText.implicitHeight
+					
+					color:				enabled ? jaspTheme.white : jaspTheme.whiteBroken
```

Comment:
This enabled does not work. It should be languageModel.useAlternativeLocale instead. But I don't think it should be grayed out if the alternative locale is not checked: the way the numbers are written is still depending on the language.